src/Storage.js

```diff
@@ -0,0 +1,69 @@
+import redis from 'redis'
+
+class Storage {
```

Comment:
We should indicate the functions that need to be implemented. I think maybe renaming it ChallengeStorage is the best way and having simple (`get`, `set`, `clear`) functions.